### PR TITLE
PMM-9795 get saas host

### PR DESCRIPTION
**Pull request by fabio-silva**
**What this PR does / why we need it**: Create an endpoint on Grafana's backend to read and retrieve env var `PERCONA_SAAS_HOST `. This is then used to save the SAAS host on the store (i.e. dev or prod).

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-9795

### CODE REVIEW

**Review by nicolalamacchia — APPROVED** [review]

File: public/app/percona/settings/components/Platform/Connect/Connect.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const Connect: FC = () => {
           showErrorOnBlur
           required
         />
-        <a href={`${process.env.PERCONA_SAAS_HOST}/profile`} rel="noreferrer" target="_blank">
+        <a href={`${saasHost}/profile`} rel="noreferrer" target="_blank">
```
Comment (atymchuk) [inline public/app/percona/settings/components/Platform/Connect/Connect.tsx:69]:
```suggestion
        <a href={`${saasHost}/profile`} rel="noreferrer, noopener" target="_blank">
```

**Review by denisok — APPROVED** [review]

File: public/app/percona/settings/components/Platform/Connect/Connect.tsx (hunk 1 above)
Comment (fabio-silva) [reply public/app/percona/settings/components/Platform/Connect/Connect.tsx]: 👍

**Review by filipmikes1 — APPROVED** [review]

File: public/app/percona/shared/components/PerconaBootstrapper/PerconaBootstrapper.tsx [hunk 2]
```diff
@@ -39,9 +41,16 @@ export const PerconaBootstrapper = () => {
       }
     };
 
-    getSettings();
-    getUserStatus();
-    dispatch(fetchServerInfoAction());
+    const bootstrap = async () => {
+      await getSettings();
```
Comment (atymchuk) [inline public/app/percona/shared/components/PerconaBootstrapper/PerconaBootstrapper.tsx:45]: Do we handle errors inside those methods or do we want to do it here?

File: public/app/percona/settings/components/Platform/Connect/Connect.tsx (hunk 1 above)
Comment (atymchuk) [reply public/app/percona/settings/components/Platform/Connect/Connect.tsx]: See why https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

File: public/app/percona/shared/components/PerconaBootstrapper/PerconaBootstrapper.tsx (hunk 2 above)
Comment (fabio-silva) [reply public/app/percona/shared/components/PerconaBootstrapper/PerconaBootstrapper.tsx:45]: Most of them are being handled in the reducers, using the `withSerializedError` util from Grafana.
`getSettings` and `getUserStatus` are being handled here but they should go to the reducers. I have a PR with that but I think it's better to wait for 2.27.0 before doing that change

**Pull request merged**